roles/devscripts/molecule/default/converge.yml

```diff
@@ -26,7 +26,7 @@
       domains:
         - "ocp.openstack.lab"
     cifmw_devscripts_dry_run: true
-    cifmw_devscripts_ocp_version: "4.13.12"
+    cifmw_devscripts_ocp_version: "4.14.9"
```

Comment:
Should we bump the version in the defaults?

Reply:
Let's move to the latest version in a separate PR. I am running some tests with `4.14.9` based on the results will spawn a separate PR. (easy for revert also)